src/Libraries/RevitNodes/Elements/ElevationMarker.cs

```diff
@@ -0,0 +1,162 @@
+        }
+
+        #endregion
+
+        #region Private constructors
+
+        /// <summary>
+        /// Private constructor
+        /// </summary>
+        private ElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            SafeInit(() => InitElevationMarker(marker));
+        }
+
+        #endregion
+
+        #region Helpers for private constructors
+
+        /// <summary>
+        /// Initialize a ElevationMarker element
+        /// </summary>
+        private void InitElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            InternalSetElevationMarker(marker);
+        }
+
+        #endregion
+
+        #region Private mutators
+
+        /// <summary>
+        /// Set the InternalMarker property and the associated element id and unique id
+        /// </summary>
+        /// <param name="marker"></param>
+        private void InternalSetElevationMarker(Autodesk.Revit.DB.ElevationMarker marker)
+        {
+            InternalMarker = marker;
+            InternalElementId = marker.Id;
+            InternalUniqueId = marker.UniqueId;
+        }
+
+        #endregion
+
+        #region Public static constructors
+
+        /// <summary>
+        /// Creates a new ElevationMarker.
+        /// </summary>
+        /// <param name="viewFamilyType">The ViewFamilyType that will be used by all elevations hosted on the new ElevationMarker.</param>
+        /// <param name="location">The desired origin for the ElevationMarker.</param>
+        /// <param name="initialViewScale">The view scale will be automatically applied to new elevations created on the ElevationMarker. The scale is the ratio of true model size to paper size.</param>
+        /// <returns>The new ElevationMarker element.</returns>
+        public static ElevationMarker Create(Element viewFamilyType, Autodesk.DesignScript.Geometry.Point location, int initialViewScale)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            var elevationMarker = Autodesk.Revit.DB.ElevationMarker.CreateElevationMarker(Document, viewFamilyType.InternalElement.Id, location.ToXyz(), initialViewScale);
+            TransactionManager.Instance.TransactionTaskDone();
+            return new ElevationMarker(elevationMarker);
+        }
+
+        #endregion
+
+        #region Public Methods
+
+        /// <summary>
+        /// Creates a new elevation ViewSection on the ElevationMarker at the desired index.
+        /// </summary>
+        /// <param name="planView">The PlanView in which the ElevationMarker is visible. The new elevation ViewSection will derive its extents and inherit settings from the ViewPlan.</param>
+        /// <param name="index">The index on the ElevationMarker where the new elevation ViewSection will be placed.</param>
+        /// <returns>The new elevation ViewSection.</returns>
+        public SectionView CreateElevationByMarkerIndex(Revit.Elements.Views.View planView, int index)
```

Comment:
similarly, name needs to be `ByPlanView` or `ByViewAndIndex` or along those lines